src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -56,7 +56,11 @@ public override void ExecuteCmdlet()
                             string vmScaleSetName = this.VMScaleSetName;
                             VirtualMachineScaleSet parameters = new VirtualMachineScaleSet();
                             ComputeAutomationAutoMapperProfile.Mapper.Map<PSVirtualMachineScaleSet, VirtualMachineScaleSet>(this.VirtualMachineScaleSet, parameters);
-
+                            if (parameters?.VirtualMachineProfile?.StorageProfile?.ImageReference?.Version?.ToLower() != "latest")
+                            {
+                                WriteWarning("You�are�deploying�VMSS�pinned�to�a�specific�image�version�from�Azure�Marketplace.�\n" +
```

Comment:
Please also add a message that pinned image version may cause a problem of automatic OS upgrade (or the message from the issue)

Reply:
you mean the whole message like this, right? 
You are deploying VMSS pinned to a specific image version from Azure Marketplace. Consider using \"latest\" as the image version.

Reply:
Message updated to:
"You are deploying VMSS pinned to a specific image version from Azure Marketplace. 
Consider using "latest" as the image version. This allows VMSS to auto upgrade when a newer version is available."